### ISSUE TITLE
Build fails without bash

### ISSUE THREAD

**Issue by libesz** [acassen/keepalived#1739]
**Describe the bug**
Build fails in environments where SH is the mandatory shell

**To Reproduce**
Stock alpine linux builds are failing with the current master branch, and also the Dockerfile provided by the keepalived repo fails (which is also alpine based): 
```
docker build -t temp-keepalived .
[...]
checking for ar... ar
checking the archiver (ar) interface... ar
./configure: line 5960: syntax error: unexpected redirection
The command '/bin/sh -c apk --no-cache add      gcc     ipset   ipset-dev       iptables        iptables-dev   libnfnetlink     libnfnetlink-dev        libnl3  libnl3-dev      libnftnl-dev    make    musl-dev        openssl openssl-dev     autoconf        automake     && cd /tmp/keepalived-sources     && ./build_setup     && ./configure --disable-dynamic-linking     && make && make install     && cd -     && rm -rf /tmp/keepalived-sources     && apk --no-cache del    gcc     ipset-dev       iptables-dev    libnfnetlink-dev        libnl3-dev      libnftnl-dev    make    musl-dev        openssl-dev     autoconf        automake' returned a non-zero code: 2
```

**Expected behavior**
I would expect the build to not fail, as in case of 2.1.5.

**Keepalived version**
Any source tree which contains this commit: https://github.com/acassen/keepalived/commit/895162003fb8d09229d02b1bab3f3d63c69893ad

**Distro (please complete the following information):**
 - Alpine 3.*

**Details of any containerisation or hosted service (e.g. AWS)**
-

**Configuration file:**
-

**Notify and track scripts**
-

**System Log entries**
-

**Did keepalived coredump?**
-

**Additional context**
I suspect the problem is with the `<<<` usage [here](https://github.com/acassen/keepalived/commit/895162003fb8d09229d02b1bab3f3d63c69893ad#diff-67e997bcfdac55191033d57a16d1408aR485), which is AFAIK a bash thing.
Maybe the problem is connecting to the fact that Docker enforces sh usage during image build.

**Comment by pqarmitage** [comment]
Problem reproduced on Alpine 3.11, and it is now resolved by commit 820007f. It was indeed the use of `<<<`, in two places, that caused the problem.

Many thanks for reporting the issue.

**Comment by libesz** [comment]
Thanks @pqarmitage for the fast solution!
